packages/package_info_plus/package_info_plus_linux/lib/src/package_info.dart

```diff
@@ -23,6 +23,7 @@ class PackageInfoLinux extends PackageInfoPlatform {
       buildNumber: versionJson['build_number'] ?? '',
       packageName: versionJson['package_name'] ?? '',
       buildSignature: '',
+      installerStore: null,
```

Comment:
What @vbuberen means is to not pass null to these parameters, as it is the default value.

(shouldn't that be picked by flutter analyze?)

Reply:
I got that, I'm just away from a computer until the end of the week, so I can't fix it until then.

Reply:
If my four suggestions below are ok, you (or I) can commit them via browser 🙂